llvm/lib/Analysis/ValueTracking.cpp

```diff
@@ -558,7 +558,7 @@ bool llvm::isValidAssumeForContext(const Instruction *Inv,
 // example Pred=EQ, RHS=isKnownNonZero. cmpExcludesZero is called in loops
 // so the extra compile time may not be worth it, but possibly a second API
 // should be created for use outside of loops.
-static bool cmpExcludesZero(CmpInst::Predicate Pred, const Value *RHS) {
+static bool cmpExcludesZero(CmpInst::Predicate Pred, Value *RHS) {
```

Comment:
Why does this drop the `const` qualifier?

Reply:
Added back (it was for matching `m_ImmConstant()`)